src/kaocha/specs.clj

```diff
@@ -4,8 +4,13 @@
             [expound.alpha :as expound])
   (:import (java.io FileNotFoundException)))
 
+(defn s-gen [_])
+(defn s-with-gen [spec _] spec)
+
 (try
   (require 'clojure.test.check.generators)
+  (def s-gen @(resolve 'clojure.spec.alpha/gen))
+  (def s-with-gen @(resolve 'clojure.spec.alpha/with-gen))
   (catch FileNotFoundException _))
```

Comment:
Would it make more sense to put the definitions of s-gen and s-with-gen here? I think they're functionally identical, but it might be clearer. Also, it reduces the odds of the fallback definitions being separated from the regular ones.
```suggestion
  (catch FileNotFoundException _
  (defn s-gen [_])
  (defn s-with-gen [spec _] spec)))
```

Reply:
I think having them at the top level may be beneficial for people using tools that rely on static analysis (i.e. Cursive). Not sure if it's smart enough to infer that `s-gen` and `s-with-gen` exist otherwise.